service/controller/resource/templateignition/create.go

```diff
@@ -35,35 +35,34 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	cm := &corev1.ConfigMap{
+	s := &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: key.StatusConfigMapName(cr.Spec.ClusterID),
 			Labels: map[string]string{
 				"cluster":       cr.Spec.ClusterID,
 				label.ManagedBy: project.Name(),
 			},
 		},
-		Data: map[string]string{
+		StringData: map[string]string{ // Data?
```

Comment:
Let's change this to match upstream:
https://master.cluster-api.sigs.k8s.io/developer/providers/bootstrap.html#bootstrap-secret

So only a single value called `value` of type string.

Reply:
Done